test/benchmark/lmbench-signal/result_template.json

```diff
@@ -0,0 +1,14 @@
+[
+    {
+        "name": "Average Signal handler overhead on Linux",
+        "unit": "ms",
+        "value": 0,
+        "extra": "linux_avg"
+    },
+    {
+        "name": "Average Signal handler overhead on Asterinas",
+        "unit": "ms",
```

Comment:
Use "us" as the unit

Reply:
> Use "us" as the unit

Using "us" as the unit requires multiplying the result by 1000. However, the current benchmarking design does not support this function. I suggest using the `value` field of the `result_template.json` to indicate the multiplier and this function should not be implemented in this PR.

Reply:
I see. Then let's keep using "ms".